layout/_partials/head/head-unique.njk

```diff
@@ -12,7 +12,9 @@
   {%- set canonical = canonical | replace(r/\.html$/, '') %}
 {%- endif %}
 <link rel="canonical" href="{{ canonical }}">
-
+{% if page.noindex %}
+<meta name="robots" content="noindex"/>
+{% endif %}
```

Comment:
Hi, please use `{%- ... -%}` instead of `{% ... %}`. `{% ... %}` will bring useless whitespace.

https://mozilla.github.io/nunjucks/templating.html#whitespace-control

Reply:
Hi @njzjz , thank you!  already fixed.